src/Symfony/Component/Inflector/Inflector.php

```diff
@@ -228,6 +228,9 @@ final class Inflector
         // bacteria (bacterium), criteria (criterion), phenomena (phenomenon)
         ['noi', 3, true, true, 'ions'],
 
+        // seasons (season)
+        ['nosaes', 6, true, true, 'seasons'],
```

Comment:
The issue seems to be generally related to words ending in `son`

```diff
Inflector::pluralize('treason')
- treasa
+ treasons

Inflector::pluralize('poison')
- poisa
+ poisons

Inflector::pluralize('lesson')
- lessa
+ lessons
```

Reply:
Thanks. Done